tests/test_logger.py

```diff
@@ -0,0 +1,243 @@
+# Copyright (C) 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import datetime
+import logging as deflogging  # Default logging
+
+import pytest
+
+from ansys.additive.core import LOG  # Global logger
+import ansys.additive.core.progress_logger as logger
+
+LOG_LEVELS = {"CRITICAL": 50, "ERROR": 40, "WARNING": 30, "INFO": 20, "DEBUG": 10}
+
+
+def test_global_logger_exist():
+    """Test for checking the accurate naming of the general Logger instance."""
+
+    assert isinstance(LOG.logger, deflogging.Logger)
+    assert LOG.logger.name == "PyAdditive_global"
+
+
+def test_global_logger_has_handlers():
+    """Test for checking that the general Logger has file_handlers and sdtout
+    file_handlers implemented."""
+
+    assert hasattr(LOG, "file_handler")
+    assert hasattr(LOG, "std_out_handler")
+    assert LOG.logger.hasHandlers
+    assert LOG.file_handler or LOG.std_out_handler
+
+
+def test_global_logger_logging(caplog: pytest.LogCaptureFixture):
+    """Testing the global PyAdditive logger capabilities. Forcing minimum logging level to
+    Debug, adding a message with different logging levels, checking the output and
+    restoring to original level.
+
+    Parameters
+    ----------
+    caplog : pytest.LogCaptureFixture
+        Fixture for capturing logs.
+    """
```

Comment:
I don't put doc strings in test code. I use the name of the test to describe what it does. For instance 
`def test_expected_messages_are_logged_after_setting_log_level(caplog: pytest.LogCaptureFixture):`. You're free to add doc strings if you prefer.

Reply:
I agree with making the test title self descriptive